src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import ForecastPage from "./Containers/ForecastPage";
+import HourlyForeCastPage from "./Containers/HourlyForeCastPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { WeatherStateData } from "./Context";
+
+const data = {
+  country: "Japan",
+  city: "Tokyo",
+  time: "1:00PM",
+  day: "Friday",
+  date: "22/04/2021",
+  avatar: "",
+  temperature: "22 F",
+  caption: "clear sky",
+};
```

Comment:
Why is this data here, this needs to be in constant file doesn't it?

Reply:
ok